src/profile-management/src/employees/dto/create-employee.dto.ts

```diff
@@ -1,39 +1,51 @@
 import { IsNumber, IsOptional, IsString } from 'class-validator';
-import { Role } from 'src/auth/enums/role.enum';
+import { Role } from '../../auth/enums/role.enum';
```

Comment:
hmmm i dont think that its a good idea for one service to reach out from its own folder into another services folder, maybe create a `shared` folder where you put this files which can be reached by any service

Reply:
you can also create alias in the tsconfig file, here is an example
`    "paths": {
      "@shared/*": ["src/shared/*"],
      "@modules/*": ["src/modules/*"]
    },
`

if you do it like this you can reach the those dirs with @shared so you dont need the ../../ thing

Reply:
The paths are added and now the imports are fine. The auth directory was discussed, that there is also one inside the prof management and there will be the need of a shared library in the future.